lib/exchanges/map-parsing.js

```diff
@@ -1,60 +1,36 @@
 // Function to parse the underlying maps in each exchange
-function mapParse(exchangeData, exchangeType) {
-  var highbids = exchangeData.highbids;
-  var lowasks = exchangeData.lowasks;
+function orderbookSpread(orderbook) {
 
-  // Initializing temporary variables for sorting through the maps
+  // Initializing temporary variables for sorting through the orderbooks
   var tmp_highbid = 0,
-  tmp_lowask = 1000000000000000,
+  tmp_lowask = Number.MAX_SAFE_INTEGER,
   tmp_amt_highbid = 0,
   tmp_amt_lowask = 0;
 
-  // TODO parseFloat numbers when they are stored, not when now
+  var tmp_highbid2 = 0,
+  tmp_lowask2 = Number.MAX_SAFE_INTEGER,
+  tmp_amt_highbid2 = 0,
+  tmp_amt_lowask2 = 0;
```

Comment:
Okay yeah not going to comment on all of these. Appending `2` onto a variable name tells me nothing about its purpose, don't do that.